bundler/src/timestamp.rs

```diff
@@ -19,16 +19,24 @@ pub fn determine_timestamps(
 
     // Determine timestamp via Git. Do this in parallel because it is pretty
     // slow.
-    let timestamps = paths
-        .par_iter()
-        .map(|p| {
-            if has_git {
-                timestamp_for_path_with_git(p)
```

Comment:
Wouldn't it be simpler and more accurate (in the case only a single call to git fails for some other reason, even if that seems unlikely) to call `.or_else(|_| timestamp_for_path_with_fs(p))` here?

Reply:
Yeah, thanks!